nss-day-cohort-46/rare-rest-bodacious-balloons
# Sc reactions

# Description
when you click a reaction button in postDetails, it posts to postReactions table
Fixes # (issue)

## Type of change


# Testing Instructions

```
git fetch --all
git checkout <sc-reactions>
go to postDetails
click any reaction (make sure it works when you click the image in the button too)
```

# Checklist:
- [ ] (What should I check for??) - What i said above
